scripts/misc/download_model.py

```diff
@@ -85,6 +89,9 @@ def parse_args() -> argparse.Namespace:
     download_from = args.download_from
 
     if download_from == 'http':
+        if args.tokenizer_only == True:
```

Comment:
```suggestion
        if args.tokenizer_only:
```
If we specify the type and have the default, it'll be either true or false

Reply:
yeah with the above comment, I cannot specify the type with store_action. Maybe that's safe enough? I just preferred to be explicit in case. lmk what you think

Reply:
I think it's still safe with the default value since `store_true` implies bool already so I don't think a user can provide a weird value, but not super opinionated here. I'll leave it to you!